src/zmalloc.c

```diff
@@ -673,6 +730,16 @@ size_t zmalloc_get_frag_smallbins(void) {
     return frag;
 }
 
+/* Compute the total memory wasted in fragmentation of inside small arena bins.
+ * Done by summing the memory in unused regs in all slabs of all small bins. */
+size_t zmalloc_get_frag_smallbins(void) {
+    return zmalloc_get_frag_smallbins_by_arena(UINT_MAX);
```

Comment:
MALLCTL_ARENAS_ALL is also the arena number, but it represents the maximum value.
```suggestion
    return zmalloc_get_frag_smallbins_by_arena(MALLCTL_ARENAS_ALL);
```

Reply:
ohh, i see, it is a `#define MALLCTL_ARENAS_ALL	4096`